antora-ui-camel/src/js/06-mobile-footer.js

```diff
@@ -0,0 +1,15 @@
+document.addEventListener('DOMContentLoaded', function () {
```

Comment:
Perhaps we can do a version without JavaScript? Using `<label>` for `<input type=checkbox>` and based on [`:checked` selector](https://css-tricks.com/almanac/selectors/c/checked/).

Reply:
* Well that's a creative checkbox hack and I am aware of it but I prevented using it 'cause it is a CSS hack. 
* Another point I would like to raise is even if I were to use a CSS hack with a checkbox for toggling for the footer scenario then we shouldn't use javascript for the navbar burger and dropdown arrow toggling as well, I believe.

Reply:
I think it's debatable if the using `:checked` selector and checkboxes should or should not be used. I think it's fairly common practice that got labeled a hack in the beginning and can't shake this image. For me it's about the simplicity of the solution rather than the technique, and I think in this case we do not need to use JavaScript.

Perhaps more appropriate no-JavaScript solution could be using the [`:target` CSS](https://developer.mozilla.org/en-US/docs/Web/CSS/:target) selector.

Reply:
While the `:target` selector works with toggling, it causes unnecessary scrolling in pages. I will try with the use of a `:checked` selector as well but if it works toggling with the checked selector in this scenario, I think in that way we could minimize the use of javascript for navbar burger toggling and the dropdown menu toggling for the mobile version.